src/controllers/requestController.js

```diff
@@ -51,11 +63,15 @@ const Request = {
     const { request } = req;
     if (request.status === 'pending') {
       const newRequest = await request.update({ status });
-      return sendResult(res, 200, 'updated successfully', newRequest);
+      return sendMail(req, res, newRequest);
```

Comment:
so manager will not receive a response??? because the request ends here

Reply:
The response will be sent after the mail is sent. checkout the sendmail function just below!